### PR TITLE
Test salt authentication

### PR DESCRIPTION
**Pull request by admd**
Change config to run cucumbers on correct branches.

### CODE REVIEW

File: terracumber_config/tf_files/SUSEManager-Test-NUE.tf [hunk 1]
```diff
@@ -81,8 +81,8 @@ provider "libvirt" {
 module "cucumber_testsuite" {
   source = "./modules/cucumber_testsuite"
 
-  product_version = "head"
-  
+  product_version = "4.0-released"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-Test-NUE.tf:84]: You are mixing: `Manager-4.1` branch for cucumber, and `4.0-released` for the product version.

But besides that: are you sure 1000% sure this should be `released`?

File: terracumber_config/tf_files/SUSEManager-Test-Naica-NUE.tf [hunk 2]
```diff
@@ -12,12 +12,12 @@ variable "CUCUMBER_COMMAND" {
...
 variable "CUCUMBER_GITREPO" {
   type = "string"
-  default = "https://github.com/uyuni-project/uyuni.git"
+  default = "https://github.com/SUSE/spacewalk.git"
 }
 
 variable "CUCUMBER_BRANCH" {
   type = "string"
-  default = "master"
+  default = "Manager-3.2"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-Test-Naica-NUE.tf:20]: Same

File: terracumber_config/tf_files/SUSEManager-Test-Naica-NUE.tf [hunk 3]
```diff
@@ -81,8 +81,8 @@ provider "libvirt" {
 module "cucumber_testsuite" {
   source = "./modules/cucumber_testsuite"
 
-  product_version = "head"
-  
+  product_version = "3.2-released"
```
Comment (juliogonzalez) [inline terracumber_config/tf_files/SUSEManager-Test-Naica-NUE.tf:84]: Same as before.

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
Some changes required.

And... aren't we missing a change to test 4.1?

File: terracumber_config/tf_files/SUSEManager-Test-NUE.tf (hunk 1 above)
Comment (admd) [reply terracumber_config/tf_files/SUSEManager-Test-NUE.tf:84]: ah right, 4.1 was a typo, thank you.

I don't see any problem with released as I am testing a branch based on the last released MU. Do you see any problem?

**Comment by admd** [comment]
> And... aren't we missing a change to test 4.1?

For 4.1, we already have changes in place against Hexagon

File: terracumber_config/tf_files/SUSEManager-Test-NUE.tf (hunk 1 above)
Comment (juliogonzalez) [reply terracumber_config/tf_files/SUSEManager-Test-NUE.tf:84]: I am pretty sure that if you use it, you will only get `released` packages, without the packages you want to test. Check the sumaform doc.

**Review by juliogonzalez — APPROVED** [review]
Did not notice any error, so let's give it a try.

**Pull request merged**